avocado/core/multiplexer.py

```diff
@@ -439,7 +439,7 @@ def is_parsed(self):
         """
         return self.variants is not None
 
-    def data_inject(self, key, value, path=None):
+    def data_inject(self, key, value, path=None):   # pylint: disable=E0202
```

Comment:
Can you tell what version of pylint you're using?  I'm running:

```
$ pylint --version
pylint 1.6.4, 
astroid 1.4.8
```

Which seems to be the latest version:

https://pypi.python.org/pypi/pylint

And these are the `E`rror type messages I get on `avocado/core/multiplexer.py` on master:

```
E:434,16: No value for argument 'self' in function call (no-value-for-parameter)
E:442, 4: An attribute defined in avocado.core.multiplexer line 410 hides this method (method-hidden)
E:457, 4: An attribute defined in avocado.core.multiplexer line 411 hides this method (method-hidden)
```

BTW, I don't see a problem with the changes, but I'd like to know why I'm not getting them here.